tests/models/openconfig/data/openconfig_system/parse/ios/config/result.json

```diff
@@ -0,0 +1,98 @@
+{
+  "openconfig-system:system": {
+    "config": {
+      "hostname": "csr1000v-1"
+    },
+    "clock": {
+      "config": {
+        "timezone-name": "CST -6 0"
+      }
+    },
+    "dns": {
```

Comment:
The logic of this appears to be incorrect if I'm reading this correctly. Shouldn't there be 4 DNS servers here? 
- 8.8.8.8
- 8.8.4.4
- 2001:4860:4860::8888
- 2001:4860:4860::8844

Reply:
Correct, good catch!